livekit-agents/livekit/agents/voice/events.py

```diff
@@ -152,10 +152,26 @@ class FunctionToolsExecutedEvent(BaseModel):
     function_calls: list[FunctionCall]
     function_call_outputs: list[FunctionCallOutput | None]
     created_at: float = Field(default_factory=time.time)
+    _reply_required: bool = PrivateAttr(default=False)
+    _handoff_required: bool = PrivateAttr(default=False)
 
     def zipped(self) -> list[tuple[FunctionCall, FunctionCallOutput | None]]:
         return list(zip(self.function_calls, self.function_call_outputs))
 
+    def prevent_tool_reply(self) -> None:
```

Comment:
Maybe nit, but actually I also like `cancel_tool_reply` (following the Python cancel of asyncio.Task)